app/helpers/bookings_helper.rb

```diff
@@ -0,0 +1,2 @@
+module BookingsHelper
```

Comment:
c'est quoi deja les helper je me rappelle plus ?

Reply:
we dont touch . it